optimum/habana/transformers/models/modeling_all_models.py

```diff
@@ -117,6 +117,8 @@ def gaudi_invert_attention_mask(self, encoder_attention_mask: torch.Tensor) -> t
         encoder_extended_attention_mask = encoder_attention_mask[:, None, :, :]
     if encoder_attention_mask.dim() == 2:
```

Comment:
```suggestion
    elif encoder_attention_mask.dim() == 2:
```
To make it a bit cleaner